### ISSUE TITLE
Option to reduce GPU utilization below 100%

### ISSUE THREAD

**Issue by jtcmedia** [nemosminer/DSTM-equihash-miner#4]
I use the suggestions in this thread [https://www.reddit.com/r/NiceHash/comments/66thy0/how_to_set_max_gpu_intensity_in_nicehash/](https://www.reddit.com/r/NiceHash/comments/66thy0/how_to_set_max_gpu_intensity_in_nicehash/) to limit GPU utilization so my computer is at least usable while mining. The -ct 1 option doesn't seem to work with this algorithm.

**Comment by mves001** [comment]
Strange... I can even play Stronghold Crusader HD at 1080p with 1060 card plus 7 out of 8 cores mining bytecoin on fx-8350 with xmr-stak while using nhml and equihash with dtsm in background. Also, no additional settings. Are you sure you have enough ram?

**Comment by jtcmedia** [comment]
I have a 1080ti  and 16GB of RAM so memory is not an issue. I do have a dual monitor setup.

**Comment by mves001** [comment]
Are you by any chance using MSI afterburner? Set fan speed update and hardware pooling period over or at 20000 ms. Dual monitor setup shouldn't be an issue, maybe active display port, but not that pc is unusable while mining. Also, check in nvidia settings, under power settings, set prefer maximum performance. Also, kill superfetch and windows search services. Preferably put on disable. Hopefully, windows defender is not working. Kaspersky free version is a nice replacement over defender and it puts him to sleep.

**Comment by jtcmedia** [comment]
No, not using afterburner. I do have it set at maximum performance. It's not that it is COMPLETELY unusable, just extremely sluggish to opening menus, minimizing/maximizing windows, etc. so much so it is really annoying. I have adjusted other mining algorithms to use max 90% utilization in Nicehash legacy which prevents this problem.

**Comment by mves001** [comment]
I have in my PC 3×1060 6gb over pcie hub and 1060 3gb in pcie 16x as a main card, fx-8350 CPU, 24 GB RAM, 20 GB in virtual RAM... All cards mining usually equihash. Maybe a workaround would be if you have one more available pcie 16x, get some extremely cheap VGA which will be used for everyday work while 1080 mining... since I am out of ideas...

**Comment by mves001** [comment]
http://cryptomining-blog.com/tag/p0-power-state/
Try to check this

**Comment by jtcmedia** [comment]
Great, thanks, I will take a look.

**Comment by kodeballer** [comment]
No applicable for pascal cards - overclocking will give better effects anyway instead of a p0 state.
